thoth/common/openshift.py

```diff
@@ -1608,6 +1609,43 @@ def get_workflow(
 
         return wf
 
+    def get_workflow_status_report(
+        self,
+        workflow_id: Optional[str] = None,
+        label_selector: Optional[str] = None,
+        namespace: Optional[str] = None,
+    ) -> Tuple[Dict[str, Any], int]:
+        """Get workflow status report, derived from workflow status."""
+        if workflow_id is None:
+            return {"error": "No analysis id provided"}, 400
+        try:
+            wf_status = self.get_workflow_status(
+                name=workflow_id, label_selector=label_selector, namespace=namespace
+            )
+            status = {}
+            # the Job has not started (yet) but the Workflow has been submitted
+            # if the Workflow fails, the status will contain the finished
+            # time of the workflow
+            status["status"], code = (
+                {
+                    "finished_at": wf_status.get("finishedAt"),
```

Comment:
Will these always be available not matter what phase the workflow is in?

Reply:
`finished_at` would be `None`, when the workflow is in a running state 
For ex - `{'status': {'finished_at': None, 'reason': None, 'started_at': '2020-07-23T20:59:28Z', 'state': 'Running'}} 200`